### ISSUE TITLE
setStyle does not inherit options from antPath

### ISSUE THREAD

**Issue by sghaskell** [rubenspgcavalcante/leaflet-ant-path#80]
**Type:**
- [ x ] bug
- [ ] feature
- [ ] enhancement
- [ ] question

**Environment:**
 - OS: macOS 10.13.6
 - Browser: Chrome Version 74.0.3729.157 (Official Build) (64-bit)
 - Library Version: 1.2.0

**I'm going to open a PR:**
- [ x ] yes
- [ ] no

**Description:**
When calling `setStyle` with no options, [defaults are assigned to `pause`, `delay` and `reverse`](https://github.com/rubenspgcavalcante/leaflet-ant-path/blob/master/src/plugin/components/ant-path.component.js#L206-L215) instead of inheriting options from the antPath instance. This can lead to unwanted behavior.

Creating an `antPath` instance with the following options and calling `setStyle()` will reset `delay` to `400`, `reverse` to `false` and `paused` to `false` for all layers.

```
const path = L.polyline.antPath(route, {
  "delay": 1000,
  "dashArray": [
    10,
    20
  ],
  "weight": 5,
  "color": "#0000FF",
  "pulseColor": "#FFFFFF",
  "paused": true,
  "reverse": true,
  "hardwareAccelerated": true
});

path.setStyle()
```

`setStyle` should inherit `delay`, `pause` and `reverse` from the antPath featureGroup.

**Comment by rubenspgcavalcante** [comment]
Makes sense, I'll wait for your PR, thanks again :)

**Comment by sghaskell** [comment]
Submitted PR #79

**Comment by rubenspgcavalcante** [comment]
Already merged and available on the v1.3.0, just update your local npm package to this version :)
